src/main/java/org/jboss/ejb/client/legacy/JBossEJBProperties.java

```diff
@@ -80,6 +80,7 @@ public final class JBossEJBProperties implements Contextual<JBossEJBProperties>
     private static final String DEFAULT_ENDPOINT_NAME = "config-based-ejb-client-endpoint";
 
     private static final String PROPERTY_KEY_INVOCATION_TIMEOUT = "invocation.timeout";
+    private static final String PROPERTY_KEY_SUPPRESS_TX_PROPAGATION = "transaction.propagation.suppress";
```

Comment:
This is the _legacy_ `JBossEJBProperties`, is it intended to add new configuration to the old _legacy_ configuration approach?